sxg_rs/src/lib.rs

```diff
@@ -230,6 +229,10 @@ impl SxgWorker {
                     "application/signed-exchange;v=b3".into(),
                 ),
                 ("x-content-type-options".into(), "nosniff".into()),
+                (
+                    "cache-control".into(),
+                    format!("public, max-age={}", max_age.as_secs()),
```

Comment:
On further thought, can we make this `max_age.as_secs()/4`? (Or however you represent integer division in Rust.) The idea being we want the intermediaries to have some overlapping windows of "can keep serving valid SXG" and "fetch updated expiration times".

`/4` is a bit arbitrary. (I came up with it in #407.) Perhaps `/10` or `/2` would work as well or better.

Reply:
That's a good point. Fixed now.